library/src/main/java/com/owncloud/android/lib/resources/shares/GetShareRemoteOperation.java

```diff
@@ -61,7 +61,9 @@ protected RemoteOperationResult<List<OCShare>> run(OwnCloudClient client) {
 
         // Get the response
         try {
-            get = new GetMethod(client.getBaseUri() + ShareUtils.SHARING_API_PATH + "/" + remoteId);
+            get = new GetMethod(client.getBaseUri() +
+                    ShareUtils.SHARING_API_PATH
+                    + "/" + remoteId + "?includeTags=true");
```

Comment:
`includeTags=true` should be in a constant somewhere and shared between this class and `GetSharesRemoteOperation`

Reply:
Was quickly hardcoded for testing and then forgotten…Thanks!